meson_options.txt

```diff
@@ -8,6 +8,7 @@ option('lvfs', type : 'boolean', value : true, description : 'enable LVFS remote
 option('man', type : 'boolean', value : true, description : 'enable man pages')
 option('libarchive', type : 'boolean', value : true, description : 'enable libarchive support')
 option('gudev', type : 'boolean', value : true, description : 'enable GUdev support')
+option('gusb', type : 'boolean', value : true, description : 'enable GUsb support')
```

Comment:
rather than `gusb`, maybe this should be `usb`.  And then it's `enable usb support`, and tears out everything related to usb rather than surgically the things that make references to `gusb`.

Reply:
The problem with that way is we've got a lot of references to FuUsbDevice in the various exported interfaces. This would mean the libfwupdplugin ABI/API is different depending on the compile setting, which is something I'd like to avoid.